public/components/incontext_insight/index.scss

```diff
@@ -109,17 +109,30 @@
   border: none;
 }
 
-.incontextInsightGeneratePopoverFooter{
-  padding-block: 0 !important;
+.incontextInsightGeneratePopoverFooter {
+  padding: 0 $euiSizeS !important;
   margin: 0 !important;
   border: none;
+  display: flex;
+  align-items: center;
+
+  .buttonGroupDivider {
+    height: 24px;
+    border-left: $euiBorderThin;
+    margin: 0 2px;
+  }
+
+  div {
```

Comment:
Nit: This `flex-grow: 1` will apply to all div under `.incontextInsightGeneratePopoverFooter`. How about padding `justify-content: space-between` to the outer container. Then we can remove this `flex-grow: 1`.

Reply:
Sure thing